common/src/main/java/org/keycloak/common/util/HttpPostRedirect.java

```diff
@@ -25,6 +25,7 @@
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
  * @version $Revision: 1 $
  */
+@Deprecated
```

Comment:
Since this is deprecated, Javadoc should contain `@deprecated` part with description what to use instead.